.github/workflows/ci.yml

```
@@ -173,14 +234,15 @@ jobs:
           rm deploy/dist/utils/functions/anticheat.js
 
       - name: deploy
-        uses: nogsantos/scp-deploy@master
+        uses: burnett01/rsync-deployments@7.0.2
         with:
-          src: ./deploy/*
-          host: ${{ secrets.HOST }}
-          remote: ./nypsi/
-          port: 22
-          user: ${{ secrets.USERNAME }}
-          key: ${{ secrets.SSH_KEY }}
+          switches: -avzh --partial --progress --delete
```

```suggestion
          switches: -vh --partial --progress
```

`-a` not needed, `-z` compression is probably slower

`--delete` would delete anticheat